packages/file-service/test/file.e2e-spec.ts

```diff
@@ -0,0 +1,214 @@
+import { INestMicroservice } from '@nestjs/common';
+import { MicroserviceOptions, Transport } from '@nestjs/microservices';
+import { Test, TestingModule } from '@nestjs/testing';
+import * as ProtoLoader from '@grpc/proto-loader';
+import * as GRPC from '@grpc/grpc-js';
+
+import {
+  FileProtoFile,
+  FileProto,
+  ResetProtoFile,
+  FileBucketProtoFile,
+  FileProviderProtoFile
+} from 'juno-proto';
+import { AppModule } from './../src/app.module';
+
+const { JUNO_FILE_SERVICE_FILE_PACKAGE_NAME } = FileProto;
+const TEST_SERVICE_ADDR = 'file-service:50003';
```

Comment:
Tests may be failing because this is `50003` which is the same as the. upload tests, change this to `50004`